opencti-platform/opencti-front/src/private/components/nav/LeftBar.jsx

```diff
@@ -108,22 +110,21 @@ import logoFiligranTextLight from '../../../static/images/logo_filigran_text_lig
 import useEnterpriseEdition from '../../../utils/hooks/useEnterpriseEdition';
 import useDimensions from '../../../utils/hooks/useDimensions';
 
-const SMALL_BAR_WIDTH = 55;
-const OPEN_BAR_WIDTH = 180;
-
 // Deprecated - https://mui.com/system/styles/basics/
 // Do not use it for new code.
 const useStyles = makeStyles((theme) => createStyles({
   drawerPaper: {
-    width: SMALL_BAR_WIDTH,
```

Comment:
I don't think this is a good idea to replace our constant with the exact same values.